src/sentry/static/sentry/app/views/settings/components/forms/field/index.tsx

```diff
@@ -76,6 +80,11 @@ class Field extends React.Component<Props> {
      */
     disabledReason: PropTypes.string,
 
+    /**
+     * Describes the field (displays in tooltip), requires a label to also be set.
+     */
+    description: PropTypes.string,
```

Comment:
Instead of a new prop for this, can we re-use the `help` prop? It has the same intended purpose, but just displayed differently. We can then have a prop like `showHelpInTooltip` to determine where to render it

Reply:
It would also be good to have this in storybook

Reply:
yes it's a good idea 💡